onchain/src/lib.cairo

```diff
@@ -14,6 +14,7 @@ mod quests {
     pub mod unruggable_quest;
     pub mod nft_quest;
     pub mod hodl_quest;
+    pub mod username_quest;
```

Comment:
You'll need to add the username quests tests module to the tests section in this file too, otherwise the tests wont run.